dnscontrol/dnsconfig.js

```diff
@@ -57,9 +62,11 @@ var cnames = {
     "hydra": "build03",
     "nixpkgs-update-cache": "build02",
     "nixpkgs-update-logs": "build02",
+    "nl.meet": "",
```

Comment:
@raboof Where do you want this to point to?

Reply:
could you make it a cname for `nixnl.codeberg.page`?